tests/Unit/Utilities/Test_MakeNaN.cpp

```diff
@@ -0,0 +1,59 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "tests/Unit/TestingFramework.hpp"
+
+#include <complex>
+
+#include "Utilities/MakeNaN.hpp"
+
+namespace {
+
+// according to IEEE standards, NaNs must have all 1's in the exponent
```

Comment:
Why not just use `std::isnan`? Does that not work? If you get FPE signals you can turn those off at the start of the test and back on later. I think that's clearer.

Reply:
Sorry, if you include `ErrorHandling/FloatingPointExceptions.hpp` you can call `disable_floating_point_exceptions()` at the start of the test and `enable_floating_point_exceptions()` at the end. Then you should be able to just using `std::isnan`. I think all we care about is that they're set to `NaN`, not really that we can test for the signal.

Reply:
Seems close enough to me. :+1: